tests/devices/default_qubit/test_default_qubit_native_mcm.py

```diff
@@ -647,24 +649,28 @@ def func(x, y):
 
 # pylint: disable=not-an-iterable
 @pytest.mark.jax
+@pytest.mark.parametrize("diff_method", [None, "best"])
 @pytest.mark.parametrize("postselect", [None, 1])
 @pytest.mark.parametrize("reset", [False, True])
-def test_jax_jit(postselect, reset):
+def test_jax_jit(diff_method, postselect, reset):
     """Tests that DefaultQubit handles a circuit with a single mid-circuit measurement and a
     conditional gate. A single measurement of a common observable is performed at the end."""
     jax = pytest.importorskip("jax")
```

Comment:
I don't think this is needed if you mark a test with the `jax` marker.

Reply:
Correct, I'll remove it.